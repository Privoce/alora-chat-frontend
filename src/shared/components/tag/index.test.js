import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import {
	TagComponent
} from 'shared/components';

describe('<TagComponent />', () => {
	const modelI = {
		success: true,
		text: 'teste'
	};

	const getWrapper = (model) => {
		return shallow((
			<TagComponent {...model} />
		));
	};

	it('renders correctly', () => {
		const wrapper = getWrapper(modelI);
		expect(toJson(wrapper)).toMatchSnapshot();
	});
});
