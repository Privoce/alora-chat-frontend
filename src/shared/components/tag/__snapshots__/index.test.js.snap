// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TagComponent /> renders correctly 1`] = `
<div
  className="tag fadeIn success"
>
  <LabelComponent
    alignCenter={true}
    breakWord={true}
    defaultLabel={true}
    fontSemiBold={true}
    fontSize={12}
    text="teste"
    width={18}
  />
</div>
`;
